editor/src/clj/editor/material.clj

```diff
@@ -250,11 +250,15 @@ there is no way a user can know what the generated id will be for older shaders.
     :vector-type-mat3 :mat4
     :vector-type-mat4 :mat4))
 
+(def unsupported-semantic-types
+  #{:semantic-type-bone-weights
+    :semantic-type-bone-indices})
```

Comment:
At some point we might want to have these be configurable, but as an MVP1 it's not necessary.